src/module/add.js

```diff
@@ -1,29 +1,75 @@
 const desc = document.querySelector('.inp');
-const el = document.createElement('div');
 const lists = document.querySelector('.lists');
 
 const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
 
 class Task {
-  constructor(desc, complete) {
+  constructor(desc, complete, index) {
     this.description = desc.value;
     this.complete = false;
+    this.index = tasks.length;
   }
 }
 
```

Comment:
- [ ] Please make sure that the property `index` is set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal 5). :blush:  But currently, your app does not work in the same way as you can see so make sure your fix it. I think you achieve that by making `this.index = tasks.length + 1` :+1: 
  
  [Screencast from 04-07-2023 01:39:57 AM.webm](https://user-images.githubusercontent.com/92303891/230523202-1a293b4c-7bc6-4199-9ea3-8bfbe18ce9fe.webm)

- [ ] Also, make sure that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1) :pray: